Analytics/Classes/Internal/SEGHTTPClient.h

```diff
@@ -15,6 +15,8 @@ NS_ASSUME_NONNULL_BEGIN
 @interface SEGHTTPClient : NSObject
 
 @property (nonatomic, strong) SEGRequestFactory requestFactory;
+@property (nonatomic, readonly) NSMutableDictionary<NSString *, NSURLSession *> *sessionsByWriteKey;
```

Comment:
Agreed that's true in practice, but from the isolated perspective of SEGHTTPClient, it cannot guarantee how it will be used. What's the downside of the current approach anyway?